cmd/minikube/cmd/root.go

```diff
@@ -50,13 +50,14 @@ var RootCmd = &cobra.Command{
 	Short: "Minikube is a tool for managing local Kubernetes clusters.",
 	Long:  `Minikube is a CLI tool that provisions and manages single-node Kubernetes clusters optimized for development workflows.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		setFlagsUsingViper()
```

Comment:
I refactored the version command to first set the viper var to false before running the RootCmd PersistentPreRun.  There should probably be a test for this business logic but I'm not sure how to go about it.  Any ideas @dlorenc?